projects/daily_dribble_newsletter/index.html

```diff
@@ -9,18 +9,18 @@ <h1>THE DAILY DRIBBLE</h1>
 
     <!-- Image credit: Thomas Serer https://unsplash.com/@jesusance -->
     <img src="images/stadium.jpg" />
-  </body>
-</html>
-
-<!--
-THE DAILY DRIBBLE
-
-Your daily dose of soccer stories from on and off the pitch. 
-Interviews,  breaking news and more!
+    <p>Your daily dose of soccer stories from on and and off the <br>pitch. Interviews
+    , breaking news and more!</p>
 
-ENTER YOUR EMAIL
-JOIN NOW
-
-Read our terms of service.
-
- -->
+    <form action="">
+      <label for="email"></label>
+      <input type="email" name="email" id="email" placeholder="ENTER YOUR EMAIL"><br>
+      <br>
+      <input type="button" value="JOIN NOW">
```

Comment:
These two elements (button and text input) should have the same width and height as in the design.